flavor_helpers.dart

```diff
@@ -0,0 +1,6 @@
+import 'dart:io';
+
+Future<void> copyOldFormatRealmTo(String path) async {
+  const realmBundleFile = 'flutter/realm_flutter/tests/data/realm_files/old-format.realm';
```

Comment:
I have missed this one. Lets keep packages separate. Isn't this file already available in tets/data/realm_files for the dart tests?

Reply:
I need to move to a place where the tests app can pick it up. It seemed redundant to have the it two places.

Reply:
As discussed, I will use a symlink instead